src/main.cc

```diff
@@ -751,8 +753,7 @@ StartVertexAiService(
     std::unique_ptr<triton::server::HTTPServer>* service,
     const std::shared_ptr<TRITONSERVER_Server>& server,
     triton::server::TraceManager* trace_manager,
-    const std::shared_ptr<triton::server::SharedMemoryManager>&
-        shm_manager)
+    const std::shared_ptr<triton::server::SharedMemoryManager>& shm_manager)
```

Comment:
@nv-kmcgill53 You can create another format PR to weed these out. No need to run through the CI. Once that PR is merged, rebase to the main then reviewing current will be easier.

Reply:
Created [PR for formatting](https://github.com/triton-inference-server/server/pull/4146)